### DISCUSSION TITLE
docs/research/touching-fish-2b-more-efficient

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#4] [General]
# docs/intro/

怎么样才能摸到一条大鱼呢？

https://996.ninja/docs/intro/

**Comment by WeiTaNan** [comment]
今天为下班任务平均提高了5472%

**Reply by MingGH** [reply]
太离谱啦！
![](https://img.996.ninja/ninjutsu/18968bf02c1cadda2f47ca8ebf8f8d2a.png)
